lib/dom.js

```diff
@@ -53,6 +53,40 @@ declare class FileList {
     [index: number]: File;
 }
 
+/* DataTransfer */
+
+declare class DataTransfer {
+  dropEffect: string;
+  effectAllowed: string;
+
+  items: DataTransferItemList; // readonly
+
+  setDragImage(image: Element, x: number, y: number);
```

Comment:
Oh, and return types are required for methods in `declare class`. So this kind of line should be

```JavaScript
setDragImage(image: Element, x: number, y: number): void;
```

or it won't parse